### DISCUSSION TITLE
Markdown support for Dracula theme

### DISCUSSION THREAD

**Discussion by melcz** [dracula/dracula-theme#880] [Ideas] [answered]
As I started using foam I really would like my markdown preview to match the color scheme of the VSCode theme. Is there interest in this or does someone already have a css file lying around that they want to share?

**Comment by luxonauta** [comment, marked as answer]
I'll close this discussion because of inactivity and lack of general interest, but the idea is excellent.

I encourage you to view the guidelines, tips and specifications for applying the theme in any application: https://draculatheme.com/contribute
